charts/openstack-actions-runner/templates/settings.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}
+data:
+  settings.yml: |-
+    github_organization: {{ .Values.githubOrganization }}-settings
+    cloud_nine_region: {{ .Values.cloudNineRegion }}
+    cloud_nine_tenant: {{ .Values.cloudNineTenant }}
+    python_config: {{ .Values.pythonConfigModule }}
+    runner_pool: {{ .Values.runnerPool }}
```

Comment:
I'm trusting you on the output of this file. I hope that you don't need a setting like `runner_pool: {{ .Values.runnerPool | toYaml }}`